.drone.yml

```diff
@@ -0,0 +1,199 @@
+workspace:
+  base: /var/www/owncloud
+  path: apps/twofactor_totp
+
+branches:
+  - master
+
+pipeline:
+  install-server:
+    image: owncloudci/core
+    pull: true
+    version: ${OC_VERSION}
+    db_type: ${DB_TYPE}
+    db_name: ${DB_NAME=owncloud}
+    db_host: ${DB_TYPE}
+    db_username: autotest
+    db_password: owncloud
+    when:
+      matrix:
+        NEED_SERVER: true
+
+  owncloud-log:
+    image: owncloud/ubuntu:16.04
+    detach: true
+    pull: true
+    commands:
+      - tail -f /var/www/owncloud/data/owncloud.log
+    when:
+      matrix:
+        NEED_SERVER: true
+
+  install-app:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - cd /var/www/owncloud/apps/twofactor_totp
+      - make
+      - cd /var/www/owncloud/
+      - php occ a:l
+      - php occ a:e twofactor_totp
+      - php occ a:e testing
+      - php occ a:l
+      - php occ config:system:set trusted_domains 1 --value=owncloud
+      - php occ log:manage --level 0
+    when:
+      matrix:
+        NEED_INSTALL_APP: true
+
+  fix-permissions:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - chown www-data /var/www/owncloud -R
+    when:
+      matrix:
+        NEED_INSTALL_APP: true
+
+  owncloud-coding-standard:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - make test-php-style
+    when:
+      matrix:
+        TEST_SUITE: owncloud-coding-standard
+
+  phpunit-tests:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    environment:
+    - PHP_VERSION=${PHP_VERSION}
+    - COVERAGE=${COVERAGE}
+    commands:
+    - if [ -z "${COVERAGE}" ]; then make test-php-unit; fi
```

Comment:
``Makefile`` does not have any ``test-php-unit`` so I don't expect that this will work.
We will need to add PHP unit into ``Makefile``